src/utils/fetchFunctions.ts

```diff
@@ -0,0 +1,24 @@
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { client } from './fetchClient';
+import { Comment } from '../types/Comment';
+
+export const getUsers = () => {
+  return client.get<User[]>(`/users`);
+};
+
+export const getPosts = (userId: number) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
+};
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const addComment = (newComment: Comment) => {
+  return client.post<Comment>(`/comments`, newComment);
+};
```

Comment:
Ensure that the `newComment` object is validated to prevent adding comments with empty data or spaces only. This is important to meet the checklist requirement: 'make sure you can't add movie with empty data (with spaces only)'.